core/src/test/java/com/exadel/aem/backpack/core/services/impl/PackageServiceImplTest.java

```diff
@@ -141,26 +155,37 @@ JcrPackage createPackage(final PackageInfo packageInfo,
 
 
         void verifyPackageFilters(final Node packageNode,
-                                            final List<String> expectedPaths,
-                                            final Map<String, List<String>> expectedReferencedResources) throws RepositoryException {
+                                  final List<String> expectedPaths,
+                                  final List<PathModel> expectedInitialFiltersModels,
+                                  final Map<String, List<String>> expectedReferencedResources) throws RepositoryException {
             JcrPackage jcrPackage = null;
             Type listType = new TypeToken<List<String>>() {
             }.getType();
             Type mapType = new TypeToken<Map<String, List<String>>>() {
             }.getType();
+
+            Type patModelListType = new TypeToken<List<PathModel>>() {
+            }.getType();
```

Comment:
Please see my comment above considering these line breaks

Reply:
The are auto-formatted by IntelliJ